### ISSUE TITLE
Set up ssh login to avoid password

### ISSUE THREAD

**Issue by derpyplops** [Cadenza-Labs/elk_old#9]

**Comment by derpyplops** [comment]
Actually, anyone can do this. All you need to do is to copy your ssh public key and paste it in `~/ssh/authorized_keys`. After that you should be able to get in.

**Comment by lauritowal** [comment]
Yeah, for @KayKozaronek doing so you won't need to use the password anymore.

1. Update your .ssh/config:
```
Host 216.153.50.195
  HostName 216.153.50.195
  User walt
  PreferredAuthentications publickey
  IdentityFile ~/.ssh/id_rsa # your private key, could be named differently on your machine
```

2. Copy past your public key into: ~/.ssh/authorized_keys (If it doesn't exist, create it)
Basically, just follow step 5 in here mostly: 
https://towardsdatascience.com/5-steps-setup-vs-code-for-remote-development-via-ssh-from-windows-to-linux-b9bae9e8f904

**Comment by derpyplops** [comment]
Oh, don't forget to set `ForwardAgent yes`, it's nice and convenient
